static/src/stylesheets/head.content.garnett.scss

```diff
@@ -20,7 +20,7 @@
 @import 'module/content-garnett/live-blog/_live-blog.head';
 @import 'module/_breadcrumbs';
 @import 'module/_social';
-@import 'module/_layout-hints';
+@import 'module/_layout-hints.garnett';
```

Comment:
As you are just overriding a few rules, I would instead move that module a few lines above (before all garnett-related modules), then it is easy to override on a case-by-case basis.